proposal.md

```diff
@@ -1,31 +1,31 @@
 # Proposal
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a sports score
+My bot will respond by: looking at espn.com and giving that score back
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a joke 
+My bot will respond by: tell the user a random joke
 
-When a user says:
-My bot will respond by:
+When a user says: create an ai image
+My bot will respond by: make an ai image
 
-When a user says:
-My bot will respond by:
+When a user says: set me a __ min reminder for ______
+My bot will respond by: setting that reminder and alarming the user when it is done
 
-When a user says:
-My bot will respond by:
+When a user says: when @user birthday is 
+My bot will respond by: wishing that @user a happy birthday on their birthday
```

Comment:
This is possible, but it involves holding on to information for a LONG time. We will learn how to do that later in the course, so you can come back to this. For now, come up with something easier. Maybe the user can say, "It's my birthday" and then the bot can say, "Happy birthday, [username]"